apps/label-studio/app/label_studio.py

```diff
@@ -33,29 +33,34 @@ class LabelStudioStatus(PyEnum):
 
 def main(args):
     updater = StatusUpdater()
-    updater.post_update(completed=0, phase="serving", status=LabelStudioStatus.STARTING) 
+    updater.post_update(completed=10, phase="initializing",
+            phases=["initializing","processing","serving","finished"],
+            status=LabelStudioPhase.STARTING) 
```

Comment:
this can be omitted. It's set via infra and will ignore the value passed here.

Reply:
I don't understand what will be omitted?